app/src/test/java/com/malibin/study/github/domain/repository/GithubProfileRepositoryTest.kt

```diff
@@ -0,0 +1,62 @@
+package com.malibin.study.github.domain.repository
+
+import com.google.common.truth.Truth.assertThat
+import com.malibin.study.github.domain.profile.GithubProfile
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.runBlocking
+import org.junit.jupiter.api.Test
+
+internal class GithubProfileRepositoryTest {
+    @Test
+    fun `입력된 이름에 대한 Github Profile이 반환된다`() {
+        runBlocking {
```

Comment:
아래 처럼 runBlocking을 구성해도 좋답니다 :)
```suggestion
    fun `입력된 이름에 대한 Github Profile이 반환된다`() = runBlocking {
    
```